documentation-samples/multinic/azuredeploy.json

```diff
@@ -0,0 +1,287 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vnetName": {
+      "type": "string",
+      "defaultValue": "WTestVNet",
+      "metadata": {
+        "description": "Name for the new VNet."
+      }
+    },
+    "backEndSubnetName": {
+      "type": "string",
+      "defaultValue": "BackEnd",
+      "metadata": {
+        "description": "Name for the back end subnet."
+      }
+    },
+    "backEndNSGName": {
+      "type": "string",
+      "defaultValue": "NSG-BackEnd",
+      "metadata": {
+        "description": "Name for the NSG used to allow DB access, and block Internet."
+      }
+    },
+    "remoteAccessNSGName": {
+      "type": "string",
+      "defaultValue": "NSG-RemoteAccess",
+      "metadata": {
+        "description": "Name for the NSG used to allow remote access."
+      }
+    },
+    "stdStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstoragestd",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstorageprm",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Premium_LRS",
+        "Premium_GRS",
+        "Premium_ZRS"
+      ]
+    },
+    "osType": {
+      "type": "int",
+      "defaultValue": 0,
+      "allowedValues": [
```

Comment:
Interesting… I am using the numeric value later on to decide which element in the VMsetting array to use. Can I create the array as a hashtable instead?

From: Kay Singh [mailto:notifications@github.com]
Sent: Friday, October 30, 2015 11:30 AM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Telmo Sampaio <telmos@microsoft.com>
Subject: Re: [azure-quickstart-templates] Added multiNIC documentation sample (#768)


In documentation-samples/multinic/azuredeploy.json<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f768%23discussion_r43536711&data=01%7c01%7ctelmos%40microsoft.com%7c61ce02dc2e9e4621540b08d2e1581eee%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=yp3ACcRtk4AkZRVDMSWnunB1E3vFgfsPkns5QQh1WmU%3d>:

> +        "description": "Name for simple storage account."

> +      }

> +    },

> +    "prmStorageType": {

> +      "type": "string",

> +      "defaultValue": "Premium_LRS",

> +      "allowedValues": [

> +        "Premium_LRS",

> +        "Premium_GRS",

> +        "Premium_ZRS"

> +      ]

> +    },

> +    "osType": {

> +      "type": "int",

> +      "defaultValue": 0,

> +      "allowedValues": [

Any reason to have 0 and 1 instead of Linux & Windows? User is going to look at this drop down in Portal and wonder what's 0 and 1 for osType

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f768%2ffiles%23r43536711&data=01%7c01%7ctelmos%40microsoft.com%7c61ce02dc2e9e4621540b08d2e1581eee%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=fKV38XuJMwRUPcdFY10M8jQn7oqM%2fosMtSjMMbD7RLI%3d>.